### ISSUE TITLE
Question!

### ISSUE THREAD

**Issue by IsraaSulaiman** [FACG6/FENIX-BLOG#9]
I am confused! Though you did not make any validation to posts and comments to avoid injecting any scripts, they have still been added as texts. What do you think the reason is?

**Comment by AbdallahAmmar96** [comment]
if you remember in research week 6 we explain how the injecting can be done.
we did it by replacing textContent with innerHTML to make the script work
so I didn't use innerHTML in the project to avoid injecting.
@IsraaSulaiman

**Comment by IsraaSulaiman** [comment]
Is the script stored in your database as text as well? Or should it be sanitized?

**Comment by IsraaSulaiman** [comment]
> we did it by replacing textContent with innerHTML to make the script work
> so I didn't use innerHTML in the project to avoid injecting.

You're right. But you did not use dom to render the results. Where did you use textContent in your project instead of innerHTML? You used handlebars.

**Comment by IsraaSulaiman** [comment]
Maybe the reason is that we render the page from the server side using handlebars. An assumption!
